tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "strict": true,
+    "strict": false,
```

Comment:
Out of curiosity, why are we introducing the changes to the `strict` and `noImplictAny` transpiler flag changes?

Reply:
Set that due to npm build errrors. I've changed it back but have added `strictNullChecks` and set this to false. 

Similarly, set `noImplicitAny` to false due to `Object literal's property 'details' implicitly has an 'any[]' type.` errors.